packages/sdk-browser/src/utils.ts

```diff
@@ -1,4 +1,73 @@
+import { uint8ArrayFromHexString } from "@turnkey/encoding";
+import {
+  generateP256KeyPair,
+  buildAdditionalAssociatedData,
+  compressRawPublicKey,
+} from "@turnkey/crypto";
+
 import { Buffer } from "buffer";
+import bs58check from "bs58check";
+import { AeadId, CipherSuite, KdfId, KemId } from "hpke-js";
+
+import type { EmbeddedAPIKey } from "./models";
+
+// createEmbeddedAPIKey creates an embedded API key encrypted to a target key (typically embedded within an iframe).
+// This returns a bundle that can be decrypted by that target key, as well as the public key of the newly created API key.
+export const createEmbeddedAPIKey = async (
```

Comment:
Def not a light lift  but eventually would want to add this (ideally not runtime sensitive) to @turnkey/crypto.

Reply:
totally agree. would be a non trivial amount of work as we're now acutely aware of. but soon, hopefully :)